source/shared/cpp/ObjectModel/UnknownElement.cpp

```diff
@@ -13,9 +8,19 @@ UnknownElement::UnknownElement() : BaseCardElement(CardElementType::Unknown)
 {
 }
 
+Json::Value UnknownElement::SerializeToJsonValue() const
+{
+    // Important -- we need override the type value that the base element uses for us
```

Comment:
>// Important -- we need override the type value that the base element uses for us [](start = 4, length = 81)

Is this still needed? Won't the base element now do the right thing here?

Reply:
It does now -- yes :)

---
In reply to: [251153491](https://github.com/Microsoft/AdaptiveCards/pull/2332#discussion_r251153491) [](ancestors = 251153491)